back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -225,12 +226,12 @@ def update
     end
 
     sidefx.before_update(input, current_user)
-
+    cosponsor_ids = input.cosponsors.map(&:id)
```

Comment:
@kogre I really dislike this mechanism of passing the old cosponsors to be able to make a diff between them and the new consponsors on update so I only create an activity for the new ones in the array. I especially don't like that it leaks into the patches. I re-created this based on how it was handled in the initiatives. Pointers on solving this better would be helpful.